.travis.yml

```diff
@@ -583,8 +593,8 @@ install:
       export TRAVIS_PYTHON_VERSION=3.7.2;
     fi
   - if [ "$TRAVIS_OS_NAME" == "linux" ]; then
-      if [ "$TRAVIS_PYTHON_VERSION" == "2.7" ]; then
-        export TRAVIS_PYTHON_VERSION=2.7.14;
+      if [ "$TRAVIS_PYTHON_VERSION" == "3.6" ]; then
+        export TRAVIS_PYTHON_VERSION=3.6.14;
```

Comment:
@ax3l I'll need your help finding the spack-relevant version of Py3.

Reply:
I think there should be no changed to travis needed at all, as long as we test against the internal version of spdlog to start with

Reply:
I'll revert the Python-related changes and create a seperate PR for them.